class.jetpack-connection-banner.php

```diff
@@ -134,13 +134,34 @@ public static function enqueue_connect_button_scripts() {
 		wp_enqueue_script(
 			'jetpack-connect-button',
 			Assets::get_file_url_for_environment(
-				'_inc/connect-button.js', // TODO - minify?
+				'_inc/connect-button.js',
```

Comment:
Any reason not to minify this eventually?

Reply:
that comment was removed in error. we will minify on the next iteration and i'll add an asana card!